include/userlib/syscalls.hpp

```diff
@@ -10,4 +10,17 @@ void set_led(uint8_t row, uint8_t col, bool enabled);
 
 int get_time();
 
+// We need a syscall for this because SVC will not be preempted by SysTick
+// Technically this is insecure - it's mostly for debugging
+void debug_print(const char* string);
+
+// yeah i know
```

Comment:
... jesus...

Reply:
im ok with it, its purely for us to debug. but yea it is gross